src/browser/modules/Main/Main.jsx

```diff
@@ -57,6 +58,7 @@ const Main = (props) => {
           Connection to server lost. Reconnecting...
         </WarningBanner>
       </Visible>
+      <SyncReminderBar />
```

Comment:
All the other ones are named `*Banner`, could you rename this to align to that please?